lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+  attr_accessor :patrons
+  attr_reader :name, :capacity
+  def initialize(name, int)
+    @name = name
+    @capacity = int
+    @patrons = []
+  end
+
+  def add_patron(i)
```

Comment:
What is `i` here? It is important from a user empathy standpoint that you write your code to be readable and understandable. You might know what is going on at the moment but what if you had to come back to this years later or if someone else had to read your code.